File: DevryService/Snippets/classes/classes.cpp
#include <iostream>
#include <string>
using namespace std;

class Animal
{
    public:
        string Name;
        int Age;

        Animal(string name, int age)
        {
            this->Age = age;
            this->Name = name;
        }

        void Move()
        {
            cout << this->Name << ": Moving..." << endl;
        }
};

int main()
{
    Animal animal = Animal("Dog", 4);
    animal.Move();
}

File: DevryService/Snippets/classes/classes.py
class Animal:
    def __init__(self, name, age):
        self.name = name
        self.age = age
    
    def move(self):
        print("{}: Moving...".format(self.name))

animal = Animal("Dog", 4)
animal2 = Animal("Cat", 2)

animal.move()
animal2.move()